mount/mount_linux.go

```diff
@@ -305,9 +327,10 @@ func (mounter *SafeFormatAndMount) checkAndRepairXfsFilesystem(source string) er
 }
 
 // formatAndMount uses unix utils to format and mount the given disk
-func (mounter *SafeFormatAndMount) formatAndMount(source string, target string, fstype string, options []string) error {
+func (mounter *SafeFormatAndMount) formatAndMountSensitive(source string, target string, fstype string, options []string, sensitiveOptions []string) error {
 	readOnly := false
 	for _, option := range options {
+		// No need to check sensitiveOptions since that should not contain ro
```

Comment:
unless you specifically document it cannot be provided in sensitiveoptions, check both

Reply:
Will do.